docs/LICENSE.md

```diff
@@ -0,0 +1,342 @@
+    
+    Section 7 – Other Terms and Conditions.
+        a. The Licensor shall not be bound by any additional or different terms or conditions communicated by You unless expressly agreed.
+        b. Any arrangements, understandings, or agreements regarding the Licensed Material not stated herein are separate from and independent of the terms and conditions of this Public License.
+    
+    Section 8 – Interpretation.
+        a. For the avoidance of doubt, this Public License does not, and shall not be interpreted to, reduce, limit, restrict, or impose conditions on any use of the Licensed Material that could lawfully be made without permission under this Public License.
+        b. To the extent possible, if any provision of this Public License is deemed unenforceable, it shall be automatically reformed to the minimum extent necessary to make it enforceable. If the provision cannot be reformed, it shall be severed from this Public License without affecting the enforceability of the remaining terms and conditions.
+        c. No term or condition of this Public License will be waived and no failure to comply consented to unless expressly agreed to by the Licensor.
+        d. Nothing in this Public License constitutes or may be interpreted as a limitation upon, or waiver of, any privileges and immunities that apply to the Licensor or You, including from the legal processes of any jurisdiction or authority.
+
+Creative Commons is not a party to its public licenses. Notwithstanding, Creative Commons may elect to apply one of its public licenses to material it publishes and in those instances will be considered the “Licensor.” The text of the Creative Commons public licenses is dedicated to the public domain under the CC0 Public Domain Dedication. Except for the limited purpose of indicating that material is shared under a Creative Commons public license or as otherwise permitted by the Creative Commons policies published at creativecommons.org/policies, Creative Commons does not authorize the use of the trademark “Creative Commons” or any other trademark or logo of Creative Commons without its prior written consent including, without limitation, in connection with any unauthorized modifications to any of its public licenses or any other arrangements, understandings, or agreements concerning use of licensed material. For the avoidance of doubt, this paragraph does not form part of the public licenses.
+
+Creative Commons may be contacted at creativecommons.org.
+```
+
+## License for all source code and designs
+
+All source code and designs of QUÉtudes-info, including but not limited to markup, stylesheets, and scripts, are licensed under [MPL 2.0](https://www.mozilla.org/en-US/MPL/2.0/). Note that patents and trademarks are **not** licensed under this Public License (see Section 2.3(c)).
+
+*For the prose and text license, see [License for all prose content](#license-for-all-prose-content).*
+
+### The MPL 2.0 license
+
+```
```

Comment:
Same as the license above: no "unlabeled code blocks", even if it's just text (related to markdown lint rules)

```suggestion
```text
```